src/main/resources/assets/gregtech/blockstates/large_metal_sheet.json

```diff
@@ -0,0 +1,109 @@
+{
+  "forge_marker" : 1,
+  "defaults": {
+    "model" : "minecraft:cube_all",
+    "textures" : {
```

Comment:
I'm pretty sure these blockstate jsons can be simplified.

Properties defined in `defaults` are applied on all variants, so specifying the same `model` property in each variant is redundant. You should be good to just write it once on `defaults` section.

Similarly the `textures` property in `defaults` section is also redundant because equivalent properties are defined in all variants.